ext/iconv/iconv.stub.php

```diff
@@ -47,10 +47,12 @@ function iconv_mime_decode_headers(string $headers, int $mode = 0, ?string $enco
 /** @refcount 1 */
 function iconv(string $from_encoding, string $to_encoding, string $string): string|false {}
 
+#[\Deprecated(since: '5.6', message: 'use internal_encoding, input_encoding, and output_encoding INI settings instead')]
```

Comment:
Just put 8.5 as this is usually what we do when formally deprecating something.
```suggestion
#[\Deprecated(since: '8.5', message: 'use internal_encoding, input_encoding, and output_encoding INI settings instead')]
```